chrome-extension/service.js

```diff
@@ -0,0 +1,37 @@
+/**  
+ * @param {String} linkUrl, to be analyzed
+ * @returns {String} sentiment analysis message
+ */
+async function getSentimentAnaylsisMsg(linkUrl) {
+  // TODO use regular expression 
+  const videoId = linkUrl.split('v=')[1];
```

Comment:
I have made a function that gets parses url to id using regex. its in script.js called `getVideoId` I think, can you copy it and use it over here

Reply:
Done thanks